index.html

```diff
@@ -0,0 +1,452 @@
+                <p class="animated fadeInRightBig">Understand shared interests and beliefs of your audiences. </p>
+
+                <a data-scroll class="btn btn-start animated fadeInUpBig" href="loginpage/login.html">Try now</a>
+              </div>
+            </div>
+            <div class="item" style="background-image: url(images/slider/monitor.jpg)">
+              <div class="caption">
+
+                <h1 class="animated fadeInLeftBig">Audience Profiling <span>and Statistics</span></h1>
+                <p class="animated fadeInRightBig">Enable you to have in-depth understanding of your audiences and make informed decisions</p>
+
+                <a data-scroll class="btn btn-start animated fadeInUpBig" href="loginpage/login.html">Try now</a>
+              </div>
+            </div>
+            <div class="item" style="background-image: url(images/slider/tech_health.jpg)">
+              <div class="caption">
+
+                <h1 class="animated fadeInLeftBig">Scheduled <span>messaging</span></h1>
+                <p class="animated fadeInRightBig">Aligning your message with top keywords, hashtags and use influencers during optimal online presence</p>
+
+                <a data-scroll class="btn btn-start animated fadeInUpBig" href="loginpage/login.html">Try now</a>
+              </div>
+            </div>
+          </div>
+          <a class="left-control" href="#home-slider" data-slide="prev"><i class="fa fa-angle-left"></i></a>
+          <a class="right-control" href="#home-slider" data-slide="next"><i class="fa fa-angle-right"></i></a>
+
+          <a id="tohash" href="#services"><i class="fa fa-angle-down"></i></a>
+
+        </div><!--/#home-slider-->
+        <div class="main-nav">
+          <div class="container">
+            <div class="navbar-header">
+              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+              </button>
+              <a class="navbar-brand" href="index.html">
+                <h1><img class="img-responsive" src="images/logobluedrop.png" alt="logo" height="100"></h1>
+              </a>                    
+            </div>
+            <div class="collapse navbar-collapse">
+              <ul class="nav navbar-nav navbar-right">                 
+                <li class="scroll active"><a href="#home">Home</a></li>
+                <li class="scroll"><a href="#services">Services</a></li> 
+                <li class="scroll"><a href="#about-us">About Us</a></li>                     
+                <!-- <li class="scroll"><a href="#portfolio">Portfolio</a></li>-->
+                <li class="scroll"><a href="#team">Team</a></li>
+                <!-- <li class="scroll"><a href="#blog">Blog</a></li> -->
```

Comment:
Please do not use unused code as a comment in your codebase. Unused code as comments makes your code not clean.